core/templates/domain/story/story-summary.model.ts

```diff
@@ -34,8 +34,8 @@ export interface StorySummaryBackendDict {
   'all_node_dicts': StoryNodeBackendDict[],
   // This property is optional because it is only present in the
   // story summary dict of learner dashboard page.
-  'topic_url_fragment'?: string,
-  'classroom_url_fragment'?: string
+  'topic_url_fragment'?: string | undefined,
```

Comment:
I don't think explicitly adding the union type `string | undefined` is needed since using `?:` already implicitly adds `undefined` to the type

Reply:
Yeah done, Thanks!